scipy/optimize/tests/test_optimize.py

```diff
@@ -3264,19 +3266,24 @@ def setup_method(self):
     def test_smoke(self, workers, method):
         # checks parallelised optimization output is same as serial
         workers = workers or map
+
+        jac = None
+        if method in ['Newton-CG']:
+            jac = rosen_der
```

Comment:
Why the special case for `neweton-cg`? If we use the analytical derivative, the new parallel numerical differentiation is not used from what I understand.

Reply:
For Newton-CG if `hess` is going to be estimated via finite differences then `jac` has to be analytical. Therefore the only parallelisation is happening with the numerical differentiation of `hess`

Reply:
`jac` has to be analytical for Newton-CG

Reply:
Ah alright, I was not aware of this. I am curios though: why exactly does it need an analytical gradient? Is the numerically approximated Hessian too imprecise for this particular algorithm otherwise? The code contains no comment, maybe lost to history?

Reply:
I think it'll be due to inaccuracy. A second order partial derivative for the Hessian will be a lot more inaccurate than a first order partial derivative of the gradient.